backend/src/core/config.ts

```diff
@@ -468,6 +469,11 @@ const config: Config<ConfigSchema> = convict({
     default: 'postman-email-open',
     env: 'BACKEND_SES_CONFIGURATION_SET',
   },
+  noTrackingMailConfigurationSet: {
```

Comment:
can add a comment somewhere that this is the only or chosen way we have decided to control the tracking.